site/src/patterns/message.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Message
+description: The message component in the Atlas Design System
+template: standard
+---
+
+# Message Pattern
+
+The message conversation would be used in the following manner.
+
+```html
+<section class="display-grid gap-sm">
+	<article class="display-grid">
+		<div class="message">
+			<p class="message-time">May 29, 2023, 4:50PM</p>
+			<div class="persona message-persona">
+				<figure class="persona-avatar">
+					<img src="~/src/scaffold/media/avatar.png" alt="Avatar for Jane Doe" />
+				</figure>
+			</div>
+			<div class="message-content">
+				<div class="margin-bottom-xxs">
+					<div class="message-content-header">
+						<p class="message-content-display-name">Q&A Moderator</p>
+						<details class="message-content-options popover popover-right">
+							<summary>
```

Comment:
Might be good to really fully flesh out the pattern example (this is what folks will probably copy from more).

Should summary use `button-clear.button-primary`?

Should `color-primary` go away?

Should the ul > li in the popover content go away in favor of a better menu?

Reply:
updated and add action buttons to the popover